example/cmesh/t8_cmesh_read_from_vtk.cxx

```diff
@@ -30,63 +30,74 @@ along with t8code; if not, write to the Free Software Foundation, Inc.,
 #include <t8_schemes/t8_default/t8_default_cxx.hxx>
 #include <t8_forest.h>
 
+/**
+ * Construct a cmesh read from a VTK-file type supported by our vtk-reader.
+ * Given the number of cell-values in the file we read the data again from the forest
+ * and write it into the file forest.(p)vtu . 
+ * 
+ * \param[in] prefix  The prefix of the file to read the mesh from
```

Comment:
Does the file have to end in `.ply`?
It looks like it from your comments below.

Reply:
No, it can be in (.ply, .vtp, .obj, .stl, .vtk, .g)